.gitignore

```diff
@@ -0,0 +1,5 @@
+# Default ignored files
+/shelf/
+/workspace.xml
+
+.idea
```

Comment:
why did you add .ignore file

Reply:
this file is required to avoid untracked files in git